package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d2-dataset-configuration",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
Let's go for 0.3.0. I am not gonna release this version but we can set the version now for the future release